lib/nose/enumerator.rb

```diff
@@ -16,10 +16,10 @@ def initialize(workload)
     def indexes_for_query(query)
       @logger.debug "Enumerating indexes for query #{query.text}"
 
-      range = if query.range_field.nil?
+      range = if query.range_fields.nil?
                 query.order
               else
-                [query.range_field] + query.order
+                ([query.range_fields] + query.order).flatten(1)
```

Comment:
Since `query.range_fields` is already an array, I think you can just write this as `query.range_fields + query.order`